otter/test/models/test_intents.py

```diff
@@ -153,3 +156,15 @@ def test_perform_update_error_reasons(self):
         dispatcher = self.get_dispatcher(self.get_store())
         self.assertIsNone(sync_perform(dispatcher, Effect(intent)))
         self.group.update_error_reasons.assert_called_once_with(['r1', 'r2'])
+
+    def test_modify_group_state_paused(self):
+        dispatcher = self.get_dispatcher(self.get_store())
+        r = sync_perform(dispatcher,
+                         Effect(ModifyGroupStatePaused(self.group, False)))
+        self.assertIsNone(r)
+        # Returned state has updated paused
+        modified_state = self.group.modify_state_values[-1]
```

Comment:
Do we want to add a sanity check here that `self.state != modified_state`, just to ensure that they're two different objects and not the same object that's been mutated?